packages/components/src/Tile/InformationTile/_docs/InformationTile.stories.tsx

```diff
@@ -29,6 +30,20 @@ export const Playground: Story = {
   },
 }
 
+export const InfoLabelTest: Story = {
+  render: args => (
```

Comment:
One handy feature of the modern Storybook is that whenever we're just updating prop values in a story, we can pass in the new `args` instead of redefining the render, ie:

```tsx
export const InfoLabelTest: Story = {
  args: {
    infoButtonLabel: "Testy test"
  }
}
```

Will product the same result as what we had but with less lines of code 😄 

What that's doing under the hood is passing all the storybooks args into the `InformationTile` component.

Non-essential watching but this [video walk through](https://storybook.js.org/docs/api/csf) helped me get a general understanding of what's happening in the format :)

Reply:
Done!